test/integration/protocols/dexes.spec.ts

```diff
@@ -66,30 +67,15 @@ describe.skip('Dexes', () => {
   async function liquidityTestSetup({ network, swapFee }: { network: string; swapFee?: number }): Promise<string> {
     await evm.reset({
       network,
-      skipHardhatDeployFork: true,
     });
     [sender, recipient] = await ethers.getSigners();
-    const namedAccounts = await getNamedAccounts();
-    const governorAddress = namedAccounts.governor;
-    governor = await wallet.impersonate(governorAddress);
 
-    const deterministicFactory = await ethers.getContractAt<DeterministicFactory>(
-      DeterministicFactory__factory.abi,
-      '0xbb681d77506df5CA21D2214ab3923b4C056aa3e2'
-    );
-
-    await deterministicFactory.connect(governor).grantRole(await deterministicFactory.DEPLOYER_ROLE(), namedAccounts.deployer);
+    ({ msig: governor } = await deploy('DCAHubCompanion'));
```

Comment:
I think we can remove this test file? I think it was to test that we could swap with `Paraswap` and `0x` due to the `allowanceTarget`, but since we are going to be using the swappers... Doesn't seem to make much more sense now, right?

Reply:
Will delete this file in another PR